domain-model/src/main/java/uk/gov/hmcts/cmc/domain/orders/Direction.java

```diff
@@ -0,0 +1,43 @@
+package uk.gov.hmcts.cmc.domain.orders;
+
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import uk.gov.hmcts.cmc.domain.models.CollectionId;
+
+import java.time.LocalDate;
+import java.util.List;
+
+@Getter
+@EqualsAndHashCode(callSuper = true)
+public class Direction extends CollectionId {
+
+    private DirectionParty directionParty;
+    private DirectionType directionType;
+    private DirectionHeaderType directionHeaderType;
+    private LocalDate directionActionedBy;
+    private String directionComment;
+    private List<String> extraDocuments;
+    private List<String> expertReports;
+
+    @Builder
+    public Direction(
+        String id,
+        DirectionParty directionParty,
+        DirectionType directionType,
+        DirectionHeaderType directionHeaderType,
+        LocalDate directionActionedBy,
```

Comment:
I think this is wrong `directionActionedBy` shouldn't probably be a date

Reply:
Oh, I see, you mean it has to be actioned by a certain date. I would suggest picking a different name as it seems to be referred to a person :) maybe `directionDeadline`?

Reply:
renamed to directionActionDeadline